easybuild/framework/easyblock.py

```diff
@@ -4190,6 +4197,33 @@ def print_dry_run_note(loc, silent=True):
     dry_run_msg(msg, silent=silent)
 
 
+def persists_failed_compilation_log_and_artifacts(success, application_log, silent, app, err_log_path):
+    if application_log:
+        # there may be multiple log files, or the file name may be different due to zipping
+        logs = glob.glob('%s*' % application_log)
+        print_msg(
+            "Results of the build can be found in the temporary log file(s) %s" % ', '.join(logs),
+            log=_log,
+            silent=silent
+        )
+
+        if err_log_path and not(success):
```

Comment:
code style check fails here:
```
./easybuild/framework/easyblock.py:4210:32: E275 missing whitespace after keyword
```

```suggestion
        if err_log_path and not success:

Reply:
I have made some changes and this specific edit is no longer relevant. I am following the style and avoid parenthesis around single not arguments. Can we have the test rerun?